test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -924,14 +926,11 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, IDPDiscoveryF
           return waitForWebfingerCall(test);
         })
         .then(function(test) {
-          expect(processCredsSpy.calls.count()).toBe(1);
-          expect(processCredsSpy).toHaveBeenCalledWith({
-            username: 'testuser@clouditude.net'
-          });
+          expect(processCredsSpy.calls.count()).toBe(0);
           expect(test.ac.webfinger).toHaveBeenCalled();
         });
       });
-      itp('calls async processCreds function before saving a model', function () {
+      itp('does not call async processCreds function before saving a model', function () {
```

Comment:
Don't think we need two tests here. We only worry about whether it's called or not.

Reply:
+1